images/basic/fuse/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+RUN apt-get install -y build-essential git pkg-config fuse3 libfuse3-3 libfuse3-dev strace
+
+WORKDIR /fus
+
+RUN mkdir -pv mountpoint
+RUN git clone https://github.com/libfuse/libfuse
```

Comment:
`RUN gcc -D_FILE_OFFSET_BITS=64 -Wall /usr/share/doc/libfuse3-dev/examples/passthrough.c `pkg-config fuse3 --cflags --libs` -o passthrough`

Can we avoid this git clone like this?

Reply:
The command doesn't work for me, `gcc: error: /usr/share/doc/libfuse3-dev/examples/passthrough.c`